# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/i915-diag.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.2/drivers/gpu/drm/i915/i915_pci.c.vanilla	2023-04-20 18:16:28.438301211 +0200
+++ linux-6.2/drivers/gpu/drm/i915/i915_pci.c	2023-04-20 18:20:18.780294887 +0200
@@ -1327,8 +1326,10 @@
 	if (PCI_FUNC(pdev->devfn))
 		return -ENODEV;
 
-	if (!intel_mmio_bar_valid(pdev, intel_info))
+	if (!intel_mmio_bar_valid(pdev, intel_info)) {
+		dev_err(&pdev->dev, "MMIO BAR invalid\n");
 		return -ENXIO;
+	}
 
 	/* Detect if we need to wait for other drivers early on */
 	if (intel_modeset_probe_defer(pdev))
